apps/storefront/pages/monstre/feilmeldinger.mdx

```diff
@@ -1,62 +1,110 @@
-import { Meta } from '@components';
+import { Meta, ResponsiveIframe, Image } from '@components';
 import { MenuPageLayout } from '@layouts';
 import { Figma } from 'mdx-embed';
 import { Heading, Paragraph, Alert } from '@digdir/design-system-react';
 
 <Meta
-  title='Validering og feilmeldinger'
+  title='Feilmeldinger'
   description='Slik forteller du brukeren at noe har gått galt eller mangler.'
 />
 
 export default ({ children, menu }) => (
   <MenuPageLayout
     content={children}
     data={{
-      title: 'Validering og feilmeldinger',
-      date: '5. april 2023',
+      title: 'Feilmeldinger',
+      date: '3. januar 2024',
     }}
   />
 );
 
-Mønsteret er under arbeid og krever mer innsikt og samarbeid på tvers. Sitter du på innsikt som er relevant for validering og feilmeldinger? [Ta kontakt](mailto:designsystem@digdir.no) så kan vi dele innsikt og diskutere dette mønsteret sammen!
+Det beste er om vi klarer å forhindre at feil oppstår. For å unngå at feil oppstår kan vi i forkant være tydelig på forventninger og krav, gi eksempler på korrekt utfylling, og være fleksible på formattering. Vi må la brukerne kunne ombestemme seg og alltid gi dem en vei ut.
```

Comment:
Det er best om vi klarer å hindre at det oppstår feil. Da må vi på forhånd oppgi tydelig hva brukeren må gjøre for å fylle ut feltene i skjemaet riktig. Vi må godta ulik formatering, for eksempel på felt med tall, som i telefonnummer eller datoer.  Brukerne må alltid få mulighet til å  redigere felter eller gå tilbake, og kunne pause utfyllingen eller avbryte.